apps/condo/domains/user/components/UserMenu.tsx

```diff
@@ -70,32 +98,27 @@ export const UserInfo = () => {
     )
 
     return (
-        <UserInfoWrapper>
+        <UserMenuWrapper>
             {
                 auth.isAuthenticated
                     ? (
                         <Dropdown overlay={DropdownOverlay} placement='bottomCenter'>
-                            <UserInfoContainer>
+                            <UserMenuContainer>
                                 <Space size={1}>
-                                    <Avatar
-                                        size='small'
-                                        style={{ backgroundColor: gold[4], borderRadius: '8px' }}
-                                        src={avatarUrl}
-                                        icon={!avatarUrl && <GitlabFilled style={{ color: colors.white }}/>}
-                                        alt={AvatarMessage}
-                                        className='avatar'
-                                    />
+                                    <AvatarContainer>
+                                        <UserAvatar/>
+                                    </AvatarContainer>
```

Comment:
Add a link to current user profile

Reply:
Add, please, a link also to current user name.
Now it's impossible to open current user profile page.